frontend/src/components/grantaccess/UserIdentityCard.vue

```diff
@@ -40,29 +45,46 @@ const props = defineProps<{
                     {{ props.userIdentity.firstName }}
                 </span>
             </div>
-            <div class="col-2" v-if="props.userIdentity.found">
-                <label for="lastName" class="row">
-                    Last Name
-                </label>
+            <div class="col" v-if="props.userIdentity.found">
+                <label for="lastName" class="row"> Last Name </label>
                 <span id="lastName" name="lastName" class="row">
                     {{ props.userIdentity.lastName }}
                 </span>
             </div>
-            <div class="col-6 d-flex" v-if="!props.userIdentity.found">
+            <div
+                class="col"
+                v-if="
+                    props.userIdentity.found &&
+                    props.userIdentity.businessLegalName
+                "
+            >
+                <label for="organizationName" class="row">
+                    Organization Name
+                </label>
+                <span id="organizationName" name="organizationName" class="row">
+                    {{ props.userIdentity.businessLegalName }}
+                </span>
+            </div>
+            <div
+                class="col-6 d-flex"
+                v-if="!props.userIdentity.found && props.errorMgs == ''"
```

Comment:
Curious, for "User does not exist", isn't it just need to check for "!props.userIdentity.found", why need to check "errorMgs"?

Reply:
Hi Ian, because in the permission error cases, the props.userIdentity.found is also false, but in this case, we don't want to display "User does not exist". We only want to display the permission errors.

Reply:
I see. Thanks Catherine.
Although it works, the condition still is confusing. I think it is possible, and I would not using <template> in this way, probably will separate it into two <template> with mutually-exclusive condition (like,. when there is "userIdentity" result and the other one with "errorMsg".
Duplicating the columns in this case should be ok and easier to understand.

Reply:
yeah I was thinking of that before as well, but I think the design will change anyway for display error messages, so I feel it's okay for now